src/bot.js

```diff
@@ -165,6 +165,46 @@ function addLFG(MESSAGE) {
         });
 }
 
+function endSession(message){
+    var author = message.author,
+        guild_id = message.guild.id,
+        role_id = '',
```

Comment:
It's so I can use the role_id to find the right entry in config.json. If you look inside the for loop, I am getting the role ID from the role. I then need to use that ID outside of the for loop, thus the role_id variable.